### ISSUE TITLE
Got TBeam v1.1 915Mhz works

### ISSUE THREAD

**Issue by N1obu** [richonguzman/LoRa_APRS_iGate#97]
I finally got 915Mhz to work on my TBeam v1.1
I had to modify/change the platformio.ini to use the SX1276
Can you make this change permanent in future builds?

![20240518_071609](https://github.com/richonguzman/LoRa_APRS_iGate/assets/100533255/2db672a4-bcd6-407f-a0f5-5ac9681ccf1c)
![20240518_071512](https://github.com/richonguzman/LoRa_APRS_iGate/assets/100533255/234ac696-b08b-43df-aa8a-15646e2e5772)
![20240518_071104](https://github.com/richonguzman/LoRa_APRS_iGate/assets/100533255/b72b007d-785b-4b33-a5d6-fa7aff7273db)

**Comment by richonguzman** [comment]
So I should add this mod as a board for using with 915 MHz ??

**Comment by N1obu** [comment]
Sure. It gives others in the US options for their TBeams. There are two versions. SMA on the boards use the SX1276 and U.FL on the board use the SX1262.

**Comment by wb2osz** [comment]
I would also like to see support for the SX1276.  
I tried running your application on a T-Beam v1.2 with SX1276, NEO-6M, and AXP2101.
It mostly worked.  I could configure it for my wifi LAN, and send beacons, to APRS-IS, which appeared on aprs.fi.
When I tried setting setting the LoRa frequency and enabling RX/TX, all of that stopped working.
After that, I just get the initial "LoRa APRS (IGate&Digi)" screen which does not change.
It does not attach to my wifi LAN.  It does not present its own wifi access point.
Thanks!!!

P.S.  North America, South America, Australia, and Japan allow 902-928 MHz, or a subset of that, for LoRa.
